register-autonomoustuff-repository/action.yaml

```diff
@@ -0,0 +1,18 @@
+name: register-autonomoustuff-repository
+description: register-autonomoustuff-repository
+
+inputs:
+  rosdistro:
```

Comment:
TODO: Remove this input if it's not necessary.
https://github.com/autowarefoundation/autoware-github-actions/pull/8/files#r765888962

Reply:
It was necessary for rosdep.